src/DynamoCore/Graph/Workspaces/PackageDependencyInfo.cs

```diff
@@ -146,18 +159,23 @@ internal class PackageDependencyInfo : INodeLibraryDependencyInfo
         /// </summary>
         public bool IsLoaded { get; set; }
```

Comment:
So why does this property still exist?
I don't think we're setting it correctly anymore right? 
here's 2 options:

1. obsolete it and have it return true if State == state.Loaded
2. remove it and cherry pick the removal to 2.3 before we release it.

Reply:
Yes, I will remove this. But on 2.3 we would needing this to check if is loaded or missing as we do not have these states defined in 2.3, right?

Reply:
@reddyashish you are right, lets obsolete it and set true when state == loaded.

Reply:
Obsoleted this property and modified 2 tests that were still using this property.